src/utils/sliceAnsi.js

```diff
@@ -0,0 +1,67 @@
+import isFullwidthCodePoint from 'is-fullwidth-code-point';
```

Comment:
Maybe some comments in this file. No idea what's happening

Reply:
Neither do I, this was taken from npm package, but was broken with 256bit colors so this is just a temporary solution.